WebDriverAgent

```diff
@@ -1 +1 @@
-Subproject commit 6813e2549b684379e1b6a6ee96deb2b42f4e1a85
+Subproject commit a4c4f7ed24bb54a67d29719a57d397a5c90571af
```

Comment:
Please don't update WDA like this. We update it in separate PRs so it is not buried.